backuppc.json

```diff
@@ -3,8 +3,20 @@
   "release": "11.2-RELEASE",
   "artifact": "https://github.com/freenas/iocage-plugin-backuppc.git",
   "pkgs": [
+    "lang/perl5.26",
```

Comment:
Hi,

Perl 5.26 is gone in ports, we having 5.28 as default. Also I think we dont need to hardcode it since its a dependency